### PR TITLE
fix: style improvements, debug

### PR DESCRIPTION
**Pull request by Nikko89**
Minor style fixing for Layout, NoteList components. Overflow hidden solves the problem of children of 0 width parent being displayed.

### CODE REVIEW

File: client-desktop/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "raven-client-desktop",
+  "productName": "Raven Notes Test2",
```
Comment (arol) [inline client-desktop/package.json]: Raven Notes. Get out from the test zone.

File: client-desktop/src/app/components/Layout/index.js [hunk 2]
```diff
@@ -154,11 +161,12 @@ class Layout extends Component {
   {this.renderTypeOf(Layout.NoteList)}
   </StyledNoteListPane>
   <StyledEditorPane key={3}>
+  {console.log(Layout.Editor)}
```
Comment (arol) [inline client-desktop/src/app/components/Layout/index.js]: Remove `console.log`.

Comment (arol) [reply client-desktop/src/app/components/Layout/index.js]: All of them that appear in this PR.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "raven-notes",
+  "productName": "Raven Notes Test1"
```
Comment (arol) [inline package.json]: "Raven Notes" no tests.

**Comment by arol** [comment]
☝️ this last commit is `style`, you're only deleting blank lines.

File: client-desktop/package.json (hunk 1 above)
Comment (Nikko89) [reply client-desktop/package.json]: Competely forgot about that line

**Pull request closed without merging**